frontend/gpay_agent/lib/views/login_view.dart

```diff
@@ -0,0 +1,82 @@
+import 'package:agent_app/datamodels/agent.dart';
+import 'package:agent_app/views/welcome_agent.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:google_sign_in/google_sign_in.dart';
+
+class LoginView extends StatefulWidget{
+  GoogleSignIn _googleSignIn;
+  Agent _agent;
+  @override
+  _LoginViewState createState() => _LoginViewState();
+}
+
+class _LoginViewState extends State<LoginView>{
+  GoogleSignIn _googleSignIn = GoogleSignIn(scopes: ['email']);
```

Comment:
Add a class level comment describing the class.
https://dart.dev/guides/language/effective-dart/documentation#consider-writing-a-library-level-doc-comment